pkg/analysis/numericbounds/testdata/src/a/a.go

```diff
@@ -0,0 +1,228 @@
+package a
+
+// ValidInt32WithBounds has proper bounds validation
+type ValidInt32WithBounds struct {
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	Count int32
+}
+
+// ValidInt64WithBounds has proper bounds validation
+type ValidInt64WithBounds struct {
+	// +kubebuilder:validation:Minimum=-1000
+	// +kubebuilder:validation:Maximum=1000
+	Value int64
+}
+
+// ValidInt64WithJSSafeBounds has bounds within JavaScript safe integer range
+type ValidInt64WithJSSafeBounds struct {
+	// +kubebuilder:validation:Minimum=-9007199254740991
+	// +kubebuilder:validation:Maximum=9007199254740991
+	SafeValue int64
+}
+
+// InvalidInt32NoBounds should have bounds markers
+type InvalidInt32NoBounds struct {
+	NoBounds int32 // want "field NoBounds is missing minimum bounds validation marker" "field NoBounds is missing maximum bounds validation marker"
```

Comment:
Nit, probably `minimum bound` and `maximum bound`. i.e removing the `s` (I think this is appropriate english at least)